commonutils/build.gradle

```diff
@@ -23,7 +24,7 @@ apply from: 'publish.gradle'
 
 dependencies {
     compile fileTree(dir: 'libs', include: ['*.jar'])
-    def supportVersion = '24.2.1'
+    def supportVersion = '25.3.1'
     compile "com.android.support:support-compat:$supportVersion"
```

Comment:
keeping it in 25.4 for now. Libraries doesn't need to go for latest dependency versions (especially if it's still in beta) since it will also force the consuming apps to also use the newer version (often without their knowledge))